includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -162,6 +164,19 @@ public static function add_admin_hooks() {
 		} );
 	}
 
+	/**
+	 * Enqueue style.
+	 */
+	public static function enqueue_admin_assets() {
```

Comment:
See also https://github.com/Automattic/amp-wp/pull/1414 where an `enqueue_edit_post_screen_scripts` method is introduced.